Project html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
```

Comment:
Head which contains the title of the page and the links to the CSS file and text font